config/apache.vhost-ssl.sample

```diff
@@ -5,7 +5,7 @@
 ## This configuration file assumes that mod_ssl package is installed on
 ## RHEL/CentOS hosts and it's configuration has not been modified.
 
-<VirtualHost %ssl_virtualhost_ip%:443>
+<VirtualHost %hostname(fqdn)%:443>
```

Comment:
Just now i talked with @mbukatov, He also suggested using the hostname in redirection only, not here.

Reply:
only for the redirection we need to change ip as hostname

Reply:
And what about the usage of `_default_` (or `*`) instead of particular IP here?
I see 2 benefits from that:
* the documented ssl configuration will be one step shorter (this part will not need any change against the sample configuration).
* if the Tendrl server will have two or more interfaces, the interface might be available on all of them (unless restricted on some other level).

Reply:
I agree but, it is better we support both right? when user don't want to use all IPs then out document won't help much

Reply:
I think, that using `_default_` might be better default and if user have specific requirements, he should consult it with Apache doc... But it's just my opinion...

Reply:
@mbukatov @shtripat  ^

Reply:
@GowthamShanmugam after quick discussion with @dahorak , I agree with him that using `_default_` is better

Reply:
@dahorak  explained clealry, i am also agree with this change